capnpy/compiler/request.py

```diff
@@ -77,7 +82,8 @@ def emit(self, m):
             m.w('from %s import __compiler, __schema__' % m.tmpname)
         #
         m.w('__capnpy_version__ = {version!r}', version=capnpy.__version__)
-        m.w('_check_version(__capnpy_version__)')
+        if m.version_check:
+            m.w('_check_version(__capnpy_version__)')
```

Comment:
no, I think the comment is fine. This way, if you inspect the generated schema and wonder why there is no version check, you can find the logic with a quick grep.